stellar-contract-env-common/src/env.rs

```diff
@@ -181,7 +183,7 @@ macro_rules! call_macro_with_all_host_functions {
 
             mod crypto "c" {
                 {"$_", fn compute_hash_sha256(x:Object) -> Object}
-                {"$_", fn verify_sig_ed25519(x:Object, k:Object, s:Object) -> RawVal}
+                {"$0", fn verify_sig_ed25519(x:Object, k:Object, s:Object) -> ()}
```

Comment:
All host fns should have a return value, and then if they don't have a value to return can return the Void static type.

Reply:
Fixed (sorry I had to rebase).